compiler/AST/checkAST.cpp

```diff
@@ -295,7 +295,6 @@ void checkPrimitives()
      case PRIM_RT_ERROR:
      case PRIM_RT_WARNING:
      case PRIM_NEW_PRIV_CLASS:
```

Comment:
You don't have to do this, but I wonder if we can get rid of `PRIM_NEW_PRIV_CLASS` too and just do extern calls now.

Reply:
I thought about doing that but didn't because we might later want compiler analysis to find functions calling this primitive. (Even so, we could look for calls by name...)

Reply:
True, but I think that argument holds for the privatized getter too, and that's now an extern call.

Reply:
actually it's not even an extern call.
Anyway we'll leave the issue for another PR.